am_list/runtime/queries/typescript/autometrics.scm

```diff
@@ -57,3 +57,14 @@
           (method_definition
            name: (property_identifier) @method.name)]))
  (#eq? @decorator.name "Autometrics"))
+
+((class_declaration
+  decorator: (decorator (call_expression
+                           function: (identifier) @decorator.name))
+  name: (type_identifier) @type.name
+  body: (class_body
+         [(method_signature
+           name: (property_identifier) @method.name)
+          (method_definition
+           name: (property_identifier) @method.name)]))
+ (#eq? @decorator.name "Autometrics"))
```

Comment:
This detects `@Autometrics()` decorators, while the one above only detects `@Autometrics`